int64 S_State
int32 S_ID
int64 health
int64 grass_locked

